### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJEclipse_Deploy to Azure_Screen Reader: JAWS is not reading the labels for combo-boxes

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#3375]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier!

**NOTE: Same is the case with the combo-boxes present in all other screens**

**Environment Details:**
Download and Install IntelliJ IDEA from https://docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-create-hello-world-web-app?view=azure-java-stable
IntelliJ IDEA 2019.1.3
Build #IU-191.7479.19

**OS Details: ​**
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS 1.3.1

**Repro Steps:**
1) Start JAWS and open IntelliJ IDEA application
2) Press 'Ctrl+Alt+S' to open application settings/preferences popup
3) Tab till 'Plugins' tab and press enter
4) Search for 'Azure Toolkit for IntelliJ in the Marketplace' and press enter.
5) Tab till 'Install' button and press enter
6) Restart IntelliJ IDEA application, and tab till Tools-->Azure-->Azure Sign in
7) Sign in with any Azure subscription
8) Tab till File menu and create new maven-archetype-webapp project
9) Under Project Explorer view, open and edit the file src/main/webapp/index.jsp
10) Under Project Explorer view, right-click your project, expand Azure, then click Deploy to Azure
11) Tab till the "Artifact" and "WebApp" combo-boxes and check if JAWS is reading the labels

**Actual Result:**
JAWS is not reading the labels for combo-boxes

**Expected Result:**
JAWS should read the labels for combo-boxes

**User Impact:**
Screen Reader users will not know the purpose of the combo-box if labels are not read

**MAS Reference:**
[MAS 1.3.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4%7D&cid=c1d83660-4891-4456-916d-4e23c4c6cd8a)

[Bug_3375_IntelliJ_Deploy To Azure_Combobox-Labels_JAWS.pptx](https://github.com/microsoft/azure-tools-for-java/files/3460207/Bug_3375_IntelliJ_Deploy.To.Azure_Combobox-Labels_JAWS.pptx)
[Bug_3375_IntelliJ_New WebApp_Combobox-Labels_JAWS.pptx](https://github.com/microsoft/azure-tools-for-java/files/3460208/Bug_3375_IntelliJ_New.WebApp_Combobox-Labels_JAWS.pptx)

**Comment by Flanker32** [comment]
@soumyatippapuram  This issue has been fixed in #3517, you may find the build [here](https://vscjavaci.cloudapp.net/job/Azure_Toolkits_for_Java/job/AzureToolsManager_PR/3781/Azure/), please help us verify it thanks!

**Comment by soumyatippapuram** [comment]
Regressed this issue in the above build. This issue is partially fixed. In "Deploy to Azure" pop-up, JAWS is reading the labels for "Artifact" and "WebApp" but it is not reading the label "Slot Configuration Source".
In "Create WebApp" pop-up, I am not able to navigate to the controls using keyboard.
Added attachments for reference.

[Bug_3375_JAWS_Partial Fix.pptx](https://github.com/microsoft/azure-tools-for-java/files/3600399/Bug_3375_JAWS_Partial.Fix.pptx)
[Create WebApp_Keyboard Issue.pptx](https://github.com/microsoft/azure-tools-for-java/files/3600422/Create.WebApp_Keyboard.Issue.pptx)

**Comment by Flanker32** [comment]
@soumyatippapuram Thanks for your help, the label for Slot Configuration Source has been added in #3542, you may find the build [here](https://vscjavaci.cloudapp.net/job/Azure_Toolkits_for_Java/job/AzureToolsManager_develop/1251/Azure/).

For the second issue, i can't reproduce it in my machine, both IntelliJ 2019.1 2019.2 works well, here is the gif.

![3375-2](https://user-images.githubusercontent.com/12445236/64937274-3e138b00-d88c-11e9-9bc7-a9d3d0b46c7d.gif)

Could you please verify the issue again? And see whether users could tab to other fields when create new webapp. And please also share a bit more information like the OS, IntelliJ/JDK version so that we could create another issue for the tab issues.

**Comment by soumyatippapuram** [comment]
Regressed this issue in the build mentioned above. This issue is fixed. Hence, closing this. Added attachments for reference.
[Bug_3375_Create New WebApp_Labels_JAWS_Fixed.pptx](https://github.com/microsoft/azure-tools-for-java/files/3616086/Bug_3375_Create.New.WebApp_Labels_JAWS_Fixed.pptx)
[Bug_3375_Deploy to Azure_Labels_JAWS_Fixed.pptx](https://github.com/microsoft/azure-tools-for-java/files/3616093/Bug_3375_Deploy.to.Azure_Labels_JAWS_Fixed.pptx)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749770
